pandas/_libs/intervaltree.pxi.in

```diff
@@ -97,6 +99,27 @@ cdef class IntervalTree(IntervalMixin):
             self._right_sorter = np.argsort(self.right)
         return self._right_sorter
 
+    @property
+    def is_overlapping(self):
+        """
+        Determine if the IntervalTree contains overlapping intervals.
```

Comment:
can you add that you are caching this in _is_overlapping

Reply:
done